packages/material-ui/src/CircularProgress/CircularProgress.test.js

```diff
@@ -76,25 +76,6 @@ describe('<CircularProgress />', () => {
     expect(circle).to.have.attribute('cy', '44');
   });
 
-  describe('prop: variant="static', () => {
-    it('should set strokeDasharray of circle', () => {
```

Comment:
This test should still pass. You should just stub out `console.error` (and ideally assert the prop type warning). Similar to https://github.com/mui-org/material-ui/pull/22090/files#diff-12042bd6b6c4718abf611a9e96fb1b55R201-R224 (minus the `checkPropTypes` part which is basically the `const { container } = render(<CircularProgress variant="static" value={70} />);` line in this test.